data_steward/bq_utils.py

```diff
@@ -739,6 +739,31 @@ def create_dataset(
     return insert_result
 
 
+def get_sql_row_expr(row: dict, fields: list):
+    """
+    Get appropriate SQL expression for a row based on fields spec
+
+    :param row: csv row with untyped values
```

Comment:
CSV row as in, a parsed line from a CSV file? Probably worth making this clearer maybe in the function name e.g. maybe `csv_line_to_sql_row_expr`

Reply:
I have renamed the function and clarified the docstring